README.md

```
@@ -17,79 +17,131 @@
 </p>
 
 <p align="center">
-  <!-- Product -->
-  <a href="https://app.getquriousai.xyz" target="_blank"><img alt="Static Badge" src="https://img.shields.io/badge/Product-F04438"></a>
-
-  <!-- Repo Stars -->
+  <a href="https://app.getquriousai.xyz" target="_blank">
+    <img alt="Product" src="https://img.shields.io/badge/Product-F04438">
+  </a>
   <img src="https://img.shields.io/github/stars/QuriousAI/qurious" />
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)